src/main/java/com/xpanxion/java/springboot/da1/demo/controller/student6/GymController.java

```diff
@@ -0,0 +1,25 @@
+package com.xpanxion.java.springboot.da1.demo.controller.student6;
+
+import com.xpanxion.java.springboot.da1.demo.model.student6.Gym;
+import com.xpanxion.java.springboot.da1.demo.service.student6.GymService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class GymController {
+
+    @Autowired
+    GymService gymService;
+
+    @GetMapping("student6/api/v1/gym{id}")
```

Comment:
The exercise states that we need to create this endpoint: `student{id}/api/v1/gym?name={gym name here}`